src/DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp

```diff
@@ -257,17 +271,19 @@ struct DetAndInverseImpl<Symmetry<2, 1>, Index0, Index1,
     u10 = inv_p10 - (inv_p_q10 * w00 + inv_p_q11 * w10);
     u11 = inv_p11 - (inv_p_q10 * w01 + inv_p_q11 * w11);
 
-    // det is non-const so that it can be moved into the std::pair:
-    Scalar<T> det{det_p * det_inv_x};
-    return std::make_pair(std::move(det), std::move(inv));
+    get(*det) = det_p * det_inv_x;
```

Comment:
Couldn't this become `get(*det) *= det_inv_x;` ? I assume this would be more efficient than what is written in now? Sorry I didn't catch this the first time around.